Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36295&attempt_id=fc432e57-ca0f-4502-9784-dcba2b664d89)

<details><summary>Partial log (click to expand)</summary><p>

```
      _feh_cmp_filename in filelist.o
      _feh_cmp_name in filelist.o
      _feh_cmp_dirname in filelist.o
ld: symbol(s) not found for architecture x86_64
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:44: feh] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-feh-2.25.drv-0/feh-2.25/src'
make: *** [Makefile:6: build-src] Error 2
builder for '/nix/store/dpsmkhkd7021kc0jzyqcqijq3zd9by57-feh-2.25.drv' failed with exit code 2
error: build of '/nix/store/dpsmkhkd7021kc0jzyqcqijq3zd9by57-feh-2.25.drv' failed
```
</p></details>

--- a/pkgs/applications/graphics/feh/default.nix
+++ b/pkgs/applications/graphics/feh/default.nix
@@ -6,11 +6,11 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "feh-${version}";
-  version = "2.24";
+  version = "2.25";
 
   src = fetchurl {
     url = "https://feh.finalrewind.org/${name}.tar.bz2";
-    sha256 = "148qbxkk5m7i3cxymnfwi7aikqjyfxr306dlqm9ndp6x932js5wq";
+    sha256 = "102rwi30n09l8rih6kv6bb7lhv3djklgzill4p2zag0h700yqfq6";
   };
 
   outputs = [ "out" "man" "doc" ];
@@ -20,9 +20,9 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ xorg.libX11 xorg.libXinerama imlib2 libjpeg libpng curl libexif ];
 
-  preBuild = ''
-    makeFlags="PREFIX=$out exif=1"
-  '';
+  makeFlags = [
+    "PREFIX=$(out)" "exif=1"
+  ] ++ optional stdenv.isDarwin "verscmp=0";
 
   postBuild = ''
     pushd man

